changelog/2024-03-24-conduit-0-9-0-release.md

```diff
@@ -0,0 +1,20 @@
+---
+slug: '2024-03-24-conduit-0-9-0-release'
+title: Conduit 0.9.0 release
+draft: false
+tags: [conduit, release]
+---
+
+Revolutionize your data processing with [**Conduit v0.9**](https://github.com/ConduitIO/conduit/releases/tag/v0.9.0), featuring **enhanced processors**. This release introduces powerful new capabilities that allow you to transform and manage your data more effectively.
+
+<!--truncate-->
+
+### Key Highlights
```

Comment:
The highlights are maybe a bit too generic. In the Better processors epic we rewrote the existing processors, provided docs for all of that and also included WASM processors along with a processor SDK.

Reply:
Pushing this change, let me know what you think


-   **Introduction of Standalone Processors**: Conduit 0.9 allows developers to create custom processors using Web Assembly (WASM), enabling flexibility in data processing with languages like Go, C#, Rust, and Kotlin.

-   **Enhanced Processor Framework**: The overhaul of the processor framework improves usability, allowing for more powerful and user-friendly data manipulation compared to previous versions.

-   **Comprehensive Documentation**: Detailed guides are provided for implementing standalone processors, including a "How it works" guide to assist developers in leveraging the new capabilities effectively.

-   **Improvements to Built-in Processors**: Significant enhancements have been made to existing built-in processors, making them more robust and easier to use for various data processing tasks.

-   **Dynamic Configuration**: The new connector SDK supports dynamic configuration parameters, allowing for more flexible and scalable pipeline setups.

-   **Getting Started Guide**: A user-friendly guide is available to help new users set up Conduit and explore the latest features quickly.

Reply:
LGTM!